app/routes/_gcn.circulars/circulars.server.ts

```diff
@@ -106,9 +106,65 @@ export async function search({
   totalItems: number
 }> {
   const client = await getSearch()
-
   const [startTime, endTime] = getValidDates(startDate, endDate)
 
+  const get_model_id_request = {
+    method: 'GET',
+    path: '/_ingest/pipeline',
+  }
+
+  let model_id = ''
+  try {
+    const resp = await client.transport.request(get_model_id_request)
+
+    if (resp && resp.statusCode == 200) {
+      model_id =
+        resp.body['nlp-ingest-pipeline'].processors[0].text_embedding.model_id
+    } else {
+      console.log(
+        'Error. Could not deploy model. Returned with response: ',
+        resp
+      )
+    }
+  } catch (e) {
+    console.log('Error: ', e)
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const nlpSearchQuery = query
+    ? {
+        neural: {
+          circular_embedding: {
+            query_text: query,
+            model_id,
+            k: 100,
+          },
+        },
+      }
+    : { match_all: {} }
```

Comment:
This should restore the date filtering to the nlp query
```suggestion
const nlpSearchQuery = query
    ? {
        bool: {
          must: [
            {
              neural: {
                circular_embedding: {
                  query_text: query,
                  model_id,
                  k: 100,
                },
              },
            }
          ],
          filter: {
            range: {
              createdOn: {
                gte: startTime,
                lte: endTime,
              },
            },
          },
        },
      }
    : { match_all: {} }
```